server/hedley/modules/custom/hedley_municipality/plugins/content_types/promoted_content/promoted_content.inc

```diff
@@ -74,7 +74,7 @@ function hedley_municipality_promoted_content_content_type_render($subtype, $con
 
         $items[] = [
           'title' => l($wrapper->label(), url('node/' . $wrapper->getIdentifier(), ['absolute' => TRUE])),
-          'image_url' => !empty($image) ? image_style_url('thumbnail', $image['uri']) : NULL,
+          'image_url' => !empty($image) ? image_style_url('fixed_thumbnail', $image['uri']) : NULL,
```

Comment:
oh, it has also cropping - so yeah, create a new image style -- but find a better name :)